tests/format/js/binary-expressions/__snapshots__/format.test.js.snap

```diff
@@ -1,5 +1,58 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`arrow.js - {"experimentalOperatorPosition":true} format 1`] = `
+====================================options=====================================
+experimentalOperatorPosition: true
+parsers: ["babel", "flow", "typescript"]
+printWidth: 80
+                                                                                | printWidth
+=====================================input======================================
+function f() {
+  const appEntities = getAppEntities(loadObject).filter(
+    entity => entity && entity.isInstallAvailable() && !entity.isQueue() && entity.isDisabled()
+  )
+}
+
+function f2() {
+  const appEntities = getAppEntities(loadObject).map(
+    entity => entity && entity.isInstallAvailable() && !entity.isQueue() && entity.isDisabled() && {
+      id: entity.id
+    }
+  )
+}
+
+((x) => x) + '';
+'' + ((x) => x);
+
+=====================================output=====================================
+function f() {
+  const appEntities = getAppEntities(loadObject).filter(
+    (entity) =>
+      entity &&
```

Comment:
The flag isn't implemented yet as of this commit, so these snapshots just duplicate the default behavior. A later commit that implements the flag's logic will update these tests so the specific changes are easier to see.